apps/ledger-live-desktop/src/renderer/components/WebPlatformPlayer/index.tsx

```diff
@@ -94,6 +99,7 @@ export default function WebPlatformPlayer({ manifest, inputs, onClose, config, .
             ref={webviewAPIRef}
             customHandlers={customHandlers}
             currentAccountHistDb={currentAccountHistDb}
+            Loader={Loader}
```

Comment:
Why do we need to change the Loader for the main WebPlatformPlayer ?
The WebPTXPlayer should already be correctly setup right ?